src/main/java/org/myrobotlab/service/Pir.java

```diff
@@ -18,25 +17,21 @@
 
   private static final long serialVersionUID = 1L;
 
-  boolean isActive = false;
+  String controllerName;
+
+  /**
+   * yep there are 3 states to a binary sensor true/false .... and unknown - we
+   * start with "unknown"
+   */
+  Boolean isActive = null;

```

Comment:
This should be a primitive to prevent it from being null.

Reply:
Also please rename "isActive" to just "active" ...   "is" prefix should be reserved for the getter method for a boolean of isActive()